cmd/openshift-install/log.go

```diff
@@ -82,17 +97,25 @@ func setupFileHook(baseDir string) func() {
 	if err != nil {
 		logrus.Fatal(errors.Wrap(err, "failed to open log file"))
 	}
-
 	originalHooks := logrus.LevelHooks{}
 	for k, v := range logrus.StandardLogger().Hooks {
 		originalHooks[k] = v
 	}
-	logrus.AddHook(newFileHook(logfile, logrus.TraceLevel, &logrus.TextFormatter{
+
+	fileHook := newFileHook(logfile, logrus.TraceLevel, &logrus.TextFormatter{
 		DisableColors:          true,
 		DisableTimestamp:       false,
 		FullTimestamp:          true,
 		DisableLevelTruncation: false,
-	}))
+	})
+
+	config, err := readLogConfigFile(baseDir)
+	if err == nil {
+		fileHook.config = config
+	} else {
+		logrus.Debugf("failed to parse log-config.yaml")
```

Comment:
- nit: no need for `Debugf` if there's no interpolation
- Can we emit the err?
```suggestion
		logrus.WithError(err).Debug("failed to parse log-config.yaml")
```